src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -32,10 +34,9 @@ public sealed class EmitOptions : IEquatable<EmitOptions>
         public bool IncludePrivateMembers { get; private set; }
 
         /// <summary>
-        /// If not empty then compiler should enable instrumentation.
-        /// The string specifies instrumentation settings.
+        /// Type of instrumentation that should be added to the output binary.
         /// </summary>
-        public string Instrument { get; private set; }
+        public ImmutableArray<InstrumentationKind> InstrumentationKinds { get; private set; }
```

Comment:
>ImmutableArray<InstrumentationKind> [](start = 15, length = 35)

Could this just be a bitmask or do we expect to have more than 32/64 different kinds?